eoc_journal/api_client.py

```diff
@@ -0,0 +1,231 @@
+"""
+A wrapper around the Edx API.
+"""
+
+from django.conf import settings
+
+import requests
+
+
+# Points for social activities
+SOCIAL_METRICS_POINTS = {
+    'num_threads': 10,
+    'num_comments': 15,
+    'num_replies': 15,
+    'num_upvotes': 25,
+    'num_thread_followers': 5,
+    'num_comments_generated': 15,
+}
+
+PROGRESS_IGNORE_COMPONENTS = [
```

Comment:
@bradenmacdonald unfortunately, it doesn't.

I tried getting this from `django.conf.settings`, but @mtyaka pointed out this is not available in the xblock.

I could not think of a better solution for this.